Tribler/Core/TorrentDef.py

```diff
@@ -790,6 +790,9 @@ def get_files(self, exts=None):
     def get_files_as_unicode(self, exts=None):
         return []
 
+    def get_files_as_unicode_with_length(self, exts=None):
```

Comment:
Yeah this one might look confusing, I agree. I've added a piece of comment to the top of the `TorrentDefNoMetainfo` class, explaining why this class is here and why these methods are looking so 'strange'.